templates/status.html

```diff
@@ -66,6 +69,27 @@
             var signup= "{{signup}}".split(" ");
             var logos = JSON.parse({{ logos|tojson|safe }});
 
+            var real_name = {{real_name}}
+            // Modal for telling user to not to use real names
+            if(real_name == 'yes'){
```

Comment:
Add space before `(` and before `{`

```suggestion
            if (real_name == 'yes') {
```

Reply:
Done